docs/course/1.freshman/1.blockchain.mdx

```diff
@@ -0,0 +1,51 @@
+import LucidDiagram from '@site/src/components/LucidDiagram';
+
+# What is a blockchain?
+
+Blockchains are secure, decentralized computer systems that anyone can use to keep track of data and run programs. Because they are decentralized, developers can use them without worrying about who is maintaining the system and whether they will act honestly. Instead of a single authority maintaining the system, a large network of independent computers called nodes ensure that the system is tamper-proof and stable.
+
+People use blockchains to manage data that must be persistent and protected from third-party meddling, which is why a major use of blockchains is to keep track of currency balances. Because the data is spread across many nodes that follow strict rules, users can be confident that no other user can manipulate the system, change their currency balances, or refuse valid transactions.
+
+Developers also use blockchains to host programs called smart contracts. Like currency balances, smart contracts are spread across the nodes in the system. Therefore, when you write and deploy a smart contract, you can be confident that it will run without having to worry about which computer will run it or whether someone else will try to stop it or change it.
+
+The blockchain data itself is stored in a series of blocks of computer information, each linked to the previous block in the chain by a cryptographic hash. This link makes the blockchain tamper-evident, because if the contents of a block change, the hash no longer matches the pervious block.
```

Comment:
```suggestion
The blockchain data itself is stored in a series of blocks of computer information, each linked to the previous block in the chain by a cryptographic hash. This link makes the blockchain tamper-evident, because if the contents of a block change, the hash no longer matches the previous block.
```